packages/runtimes/js/src/helpers/browser/js-loader.js

```diff
@@ -18,10 +18,13 @@ module.exports = cacheLoader(function loadJSBundle(bundle) {
     script.type = 'text/javascript';
     script.charset = 'utf-8';
     script.src = bundle;
-    script.onerror = function (e) {
+    script.onerror = function(e) {
+      var error = new TypeError(
+        `Failed to fetch dynamically imported module: ${bundle}.`,
+      );
```

Comment:
Should we somehow attach the original error so people can determine why it failed?

Reply:
We can do something like: 
```js
var error = new TypeError(
    `Failed to fetch dynamically imported module: ${bundle}. Error: ${e.message}`,
);
```
What do you think?

Reply:
Should be fixed now.